projects/packages/videopress/src/client/admin/components/video-list/index.tsx

```diff
@@ -63,6 +64,9 @@ const VideoList = ( {
 				) }
 			</div>
 			{ videos.map( ( video, index ) => {
+				const isPrivate =
```

Comment:
We can start to think about having a central place where we flag the video as private based on whatever properties we may consider, because we handling this privacy things on a lot of places. But not today :)

Reply:
Yes!
The `video.isPrivate`, as it is now, is misleading.